master.yaml

```diff
@@ -23,6 +23,11 @@ Parameters:
         Description: Key Pair for access to bastion and ecs host instances
         Type: String
 
+    PublicAlbAcmCertificate:
+        AllowedPattern: ^$|(arn:aws:acm:)([a-z0-9/:-])*([a-z0-9])$
+        Description: '[ Optional ] The AWS Certification Manager certificate ARN for the ALB certificate - this certificate should be created in the region you wish to run the ALB and must reference the Drupal domain name you use below.'
+        Type: String
```

Comment:
This is unnecessary since `master.yaml` is the place where we provide values for the parameters in templates.